content/01.main_content.md

```diff
@@ -1,35 +1,38 @@
-Recognition can greatly affect the trajectory of a scientist’s career since each form of recognition paves the way to future accolades.
-  Regardless of an individual’s merit, biases can skew which scientists are recognized.
-  Audits can combat this by identifying overlooked groups and holding the recognizing body accountable.
+Recognition by peers and the general public can greatly affect the trajectory of a scientist’s career.
+Any recognition--inclusion of one’s name in print as an expert, an invited lecture, or an award--paves the way to future accolades.
+Regardless of an individual’s merit, biases can skew which scientists are recognized.
   This is exemplified in a recent study of disparities in scientific journalism, which found that predicted East Asian names were under-quoted and under-mentioned [@doi:10.1101/2021.06.21.449261].
```

Comment:
I'm torn on the use of "predicted" here, because saying it that way before we talk about computational predictions seems out of place, but we also don't want to just say "East Asian" since that undercuts the point of the paper. Curious for your thoughts.

Reply:
I agree. We could kinda soften it by saying "names that are predicted to be of East Asian origin" ?